lib/client/config_e2e_test.go

```diff
@@ -89,6 +89,89 @@ var _ = testutils.E2eDatastoreDescribe("with config option API tests", testutils
 		Expect(cs).To(Equal(client.ConfigLocationGlobal))
 	})
 
+	It("should handle node IP in IP tunnel address", func() {
```

Comment:
Right, so this fails with KDD because KDD gets the tunnel address from the Node.PodCIDR field, and we needed to move it to an etcd-only set of tests, yeah?  Just making sure I understand.